ClosedXML/Excel/PivotTables/XLPivotTable.cs

```diff
@@ -80,6 +81,122 @@ public IEnumerable<IXLPivotField> ImplementedFields
 
         public XLPivotTableTheme Theme { get; set; }
 
+        public IXLPivotTable CopyTo(IXLCell targetCell)
+        {
+            var addressComparer = new XLAddressComparer(ignoreFixed: true);            
+            if (addressComparer.Equals(targetCell.Address, this.TargetCell.Address))
+                throw new InvalidOperationException("Cannot copy pivot table to the target cell.");
+
+            var targetSheet = targetCell.Worksheet;
+
+            var pivotTableName = this.Name;
+
+            int i = 0;
+            while (!XLHelper.ValidateName("pivot table", pivotTableName, "", targetSheet.PivotTables.Select(pvt => pvt.Name), out String message))
```

Comment:
Since we do not use `message`, better use a discard (`out var _`)

Reply:
Fixed.